bgpd/bgp_table.h

```diff
@@ -104,6 +104,7 @@ struct bgp_node {
 #define BGP_NODE_SELECT_DEFER           (1 << 4)
 #define BGP_NODE_FIB_INSTALL_PENDING    (1 << 5)
 #define BGP_NODE_FIB_INSTALLED          (1 << 6)
+#define BGP_NODE_LABEL_REQUESTED (1 << 7)
```

Comment:
let's get this aligned with code around it

Reply:
Yeah clang-format screwed it will fix up

Reply:
fixed.